backend/src/main.ts

```diff
@@ -0,0 +1,36 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+import { LoggerService } from '@tazama-lf/frms-coe-lib';
+import { validateProcessorConfig } from '@tazama-lf/frms-coe-lib/lib/config';
+import { ValidationPipe } from '@nestjs/common';
+
+async function bootstrap() {
+  // Validate and get the processor config for LoggerService
+  const configuration = validateProcessorConfig();
+  const logger = new LoggerService(configuration);
+
+  const app = await NestFactory.create(AppModule, {
+    logger,
+  });
```

Comment:
Your implementation creates another instance of the loggerService here. Might I suggest doing it like:
```suggestion
import { ValidationPipe } from '@nestjs/common';
import 'dotenv/config';

async function bootstrap() {
  const app = await NestFactory.create(AppModule);
  const logger = app.get(LoggerService);
  app.useLogger(logger);
```

Also, please make sure to install and import the `dotenv` package. Otherwise the loggerService might cause issues as NestJS handles configs a bit differently.